_episodes/02-data-management.md

```diff
@@ -9,119 +9,148 @@ objectives:
+> 5    Africa Burkina Faso       543.2552       617.1835       722.5120       794.8266       854.7360
+> 6    Africa      Burundi       339.2965       379.5646       355.2032       412.9775       464.0995
+>   gdpPercap_1977 gdpPercap_1982 gdpPercap_1987 gdpPercap_1992 gdpPercap_1997 gdpPercap_2002 gdpPercap_2007
+> 1      4910.4168      5745.1602      5681.3585      5023.2166      4797.2951      5288.0404      6223.3675
+> 2      3008.6474      2756.9537      2430.2083      2627.8457      2277.1409      2773.2873      4797.2313
+> 3      1029.1613      1277.8976      1225.8560      1191.2077      1232.9753      1372.8779      1441.2849
+> 4      3214.8578      4551.1421      6205.8839      7954.1116      8647.1423     11003.6051     12569.8518
+> 5       743.3870       807.1986       912.0631       931.7528       946.2950      1037.6452      1217.0330
+> 6       556.1033       559.6032       621.8188       631.6999       463.1151       446.4035       430.0707
+>   lifeExp_1952 lifeExp_1957 lifeExp_1962 lifeExp_1967 lifeExp_1972 lifeExp_1977 lifeExp_1982 lifeExp_1987
+> 1       43.077       45.685       48.303       51.407       54.518       58.014       61.368       65.799
+> 2       30.015       31.999       34.000       35.985       37.928       39.483       39.942       39.906
+> 3       38.223       40.358       42.618       44.885       47.014       49.190       50.904       52.337
+> 4       47.622       49.618       51.520       53.298       56.024       59.319       61.484       63.622
+> 5       31.975       34.906       37.814       40.697       43.591       46.137       48.122       49.557
+> 6       39.031       40.533       42.045       43.548       44.057       45.910       47.471       48.211
+>   lifeExp_1992 lifeExp_1997 lifeExp_2002 lifeExp_2007 pop_1952 pop_1957 pop_1962 pop_1967 pop_1972 pop_1977
+> 1       67.744       69.152       70.994       72.301  9279525 10270856 11000948 12760499 14760787 17152804
+> 2       40.647       40.963       41.003       42.731  4232095  4561361  4826015  5247469  5894858  6162675
+> 3       53.919       54.777       54.406       56.728  1738315  1925173  2151895  2427334  2761407  3168267
+> 4       62.745       52.556       46.634       50.728   442308   474639   512764   553541   619351   781472
+> 5       50.260       50.324       50.650       52.295  4469979  4713416  4919632  5127935  5433886  5889574
+> 6       44.736       45.326       47.360       49.580  2445618  2667518  2961915  3330989  3529983  3834415
+>   pop_1982 pop_1987 pop_1992 pop_1997 pop_2002 pop_2007
+> 1 20033753 23254956 26298373 29072015 31287142 33333216
+> 2  7016384  7874230  8735988  9875024 10866106 12420476
+> 3  3641603  4243788  4981671  6066080  7026113  8078314
+> 4   970347  1151184  1342614  1536536  1630347  1639131
+> 5  6634596  7586551  8878303 10352843 12251209 14326203
+> 6  4580410  5126023  5809236  6121610  7021078  8390505
+> ~~~
+> {: .output}
```

Comment:
Is it worth remarking on the line-wrapping? Or will learners know all about that by this point?